### DISCUSSION TITLE
Pre release version 3.7.0 release candidate 2

### DISCUSSION THREAD

**Discussion by softins** [jamulussoftware/jamulus#1279] [Announcements]
THIS IS A PRE RELEASE!

We have just released the Release Candidate 2 of Jamulus 3.7.0 and would be happy if you downloaded and tested it for serious bugs (server crashes etc.) before 3.7.0 gets released finally. This is expected to be the final RC before release.

View the Release Notes and get the latest release candidate here: https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0rc2

**Comment by jujudusud** [comment]
Hi all,
Testing this RC2 on Archlinux with linux-lts kernel. All is working as expected for me.

**Comment by Helondeth** [comment]
Hey,
testing on MacOS Mojave (10.14.6) on a Mac Pro.
It looks great and works very well

**Comment by ann0see** [comment]
Locked because this is outdated
